File: HackerRank/30 Days of Code/Day5_Loops.cs
// Day 5: Loops
// https://www.hackerrank.com/challenges/30-loops/problem

using System;

class Solution {
    static void Main(string[] args) {
        int n = Convert.ToInt32(Console.ReadLine());
        for (int i = 1; i <= 10; i++)
            Console.WriteLine(n + " x " + i + " = " + (n * i));
    }
}


// 반복문으로 구구단 출력

File: HackerRank/30 Days of Code/Day5_Loops.py
# Day 5: Loops
# https://www.hackerrank.com/challenges/30-loops/problem

#!/bin/python3

import math
import os
import random
import re
import sys

if __name__ == '__main__':
    n = int(input())
    for i in range(1, 11):
        print(str(n) + " x " + str(i) + " = " + str(n * i))

# 반복문으로 구구단 출력